projects/packages/my-jetpack/_inc/components/product-cards-section/backup-card/index.jsx

```diff
@@ -206,31 +205,39 @@ const NoBackupsValueSection = ( { admin, slug } ) => {
 	return (
 		<ProductCard admin={ admin } slug={ slug } showMenu isDataLoading={ isFetching }>
 			<div className={ styles[ 'no-backup-stats' ] }>
-				<div className={ styles[ 'main-stats' ] }>
-					{ sortedData.slice( 0, itemsToShow ).map( ( item, i ) => {
+				{ /* role="list" is required for VoiceOver on Safari */ }
+				{ /* eslint-disable-next-line jsx-a11y/no-redundant-roles */ }
+				<ul className={ styles[ 'main-stats' ] } role="list">
+					{ sortedData.map( ( item, i ) => {
 						const itemSlug = item[ 0 ].split( '_' )[ 1 ];
 						const value = item[ 1 ];
 
 						return (
-							<div
+							<li
 								className={ classNames( styles[ 'main-stat' ], `main-stat-${ i }` ) }
 								key={ i + itemSlug }
-								title={ getTitle( itemSlug ) }
```

Comment:
From an accessibility standpoint, the `title` attribute should be avoided:
- it's not well supported by screen readers (actual text or ARIA labels are preferred)
- while it shows a tooltip on "hover" (not instantaneous), some users may not use a mouse-like device. If the information is important to show, let's add it to the card for everybody.